pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -231,3 +231,28 @@ var UseMuxRangeFeed = settings.RegisterBoolSetting(
 	"if true, changefeed uses multiplexing rangefeed RPC",
 	false,
 )
+
+// EventConsumerWorkers specifies the maximum number of workers to use when
+// processing  events.
+var EventConsumerWorkers = settings.RegisterIntSetting(
+	settings.TenantWritable,
+	"changefeed.event_consumer_workers",
+	"the number of workers to use when processing events; 0 disables",
+	32,
```

Comment:
Seems like the improvement over single-threaded should be pretty evident with 2 workers, maybe let’s do our testing with low numbers to better understand the behavior.